docs/src/ci.md

```diff
@@ -587,6 +587,20 @@ tests:
     - npm ci
     - npx playwright test --project=$PROJECT --shard=$SHARD
 ```
+### Google Cloud Build
+
+To run Playwright tests on Google Cloud Build, use our public Docker image ([see Dockerfile](./docker.md)).
+
+```yml
+steps:
+- name: 'mcr.microsoft.com/playwright:v%%VERSION%%-jammy'
+  script: |
+    #!/usr/bin/env bash
```

Comment:
This doesn't help much the users, let's replace it with `...` as in GitLab CI?

Reply:
Done